shared/login/register/paper-key/index.js

```diff
@@ -44,7 +44,7 @@ type OwnProps = RouteProps<
 // $FlowIssue
 export default connect(
   (state: TypedState, {routeProps: {error}}: OwnProps) => ({
-    waitingForResponse: state.login.waitingForResponse,
+    waitingForResponse: state.engine.get('rpcWaitingStates').get('loginRpc'),
```

Comment:
🐑 gives a flow error. and I know `.get` isn't typesafe. Do we use getIn with flowissue elsewhere?

also it seems like .get is more likely to become typesafe that .getIn, so slight pref on keeping it as is